autoware.repos

```diff
@@ -3,146 +3,146 @@ repositories:
   core/autoware_msgs:
     type: git
     url: https://github.com/autowarefoundation/autoware_msgs.git
-    version: 1.2.0
+    version: 1.1.0  # main, Fri May 10 17:52:09 2024 +0900
```

Comment:
Are there any reasons for downgrading the version?

Reply:
Thank you @mitsudome-r san! Actually I tried to fix the version on the 1st Oct. 2024, there is no reason to use the downgraded version. Let me check with the latest version, I think the latest version would work too!